string order
string extra_prompt
---
string next_skill
string target